libSysCapabilities/src/main/ets/napi/NapiHelper.ets

```diff
@@ -41,6 +41,10 @@ export class NapiHelper {
         registerFunc('DeviceUtils.getOrientation', DeviceUtils.getOrientation);
         registerFunc('DeviceUtils.getCutoutHeight', DeviceUtils.getCutoutHeight);
         registerFunc('DeviceUtils.getCutoutWidth', DeviceUtils.getCutoutWidth);
+        registerFunc('DeviceUtils.getSafeAreaLeft', DeviceUtils.getSafeAreaLeft);
+        registerFunc('DeviceUtils.getSafeAreaTop', DeviceUtils.getSafeAreaTop);
+        registerFunc('DeviceUtils.getSafeAreaWidth', DeviceUtils.getSafeAreaWidth);
+        registerFunc('DeviceUtils.getSafeAreaHeight', DeviceUtils.getSafeAreaHeight);
```

Comment:
Why not just register a function `DeviceUtils.getSafeArea` which returns a rectangle object?